src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Api/Program.cs

```diff
@@ -20,6 +21,8 @@ public static void Main(string[] args)
                 .UseInMemoryDatabase("payment-db")
         );
         
+        builder.Services.AddSingleton(new PaymentServiceConfiguration(builder.Configuration["PaymentClientApiKey"]));
```

Comment:
Did you add this env to `variables.env`?

Reply:
This is variable I added to `variables.env` when setting `HttpClient` in CoreApp. It is the same in both apps, so there is no real need to duplicate it, is there?